src/web/src/components/CourseListing.vue

```diff
@@ -10,7 +10,7 @@
         {{ readableDate(course.date_end) }})
         <br />
         {{
-          (course.full_title && course.full_title.toUpperCase()) || course.title
+          course.title || (course.full_title && course.full_title.toUpperCase())
```

Comment:
I'm a little confused on what this is fixing, if `course.full_title` does not exist, shouldn't it default to `course.title`? Or is `course.full_title` incorrect?

Reply:
Ye same question, is there a specific course where title is more descriptive than full_title?

Reply:
The exact example given in the screenshots. The "Topics in Cognitive Science" course will always have that name in the catalog as a place holder. But the actual course name is Social Computing.

Reply:
In theory, course.full_title shouldn't be used here at all, but I wanted to leave the logic that was already in place

Reply:
course.title is course_name in the csv, i.e. the real name of the course from SIS

Reply:
Hm, so it seems like we should just use `course.title`

Reply:
That was my thought

Reply:
I'll double check what the schedule blocks use